app/controllers/admin/levelling_up_premium_payments_awards_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Admin
+  class LevellingUpPremiumPaymentsAwardsController < BaseAdminController
+    helper_method :policy_configuration
+
+    def index
+      response.headers["Content-Type"] = "text/csv"
+      response.headers["Content-Disposition"] = "attachment; filename=awards_#{academic_year}.csv"
+
+      send_data LevellingUpPremiumPayments::Award.csv_for_academic_year(academic_year), filename: "awards_#{academic_year}.csv"
```

Comment:
Super minor, wondering if those headers could be passed into send_data as parameters?